tools/docsnapper/main.rs

```diff
@@ -435,8 +448,15 @@ fn build_and_snapshot(
         ""
     };
 
+    let scale_factor = component.window().scale_factor();
+    let scale_str = if scale_factor >= 0.99 && scale_factor <= 1.01 {
```

Comment:
I'd use `approx_eq` via `euclid` - I think that's more readable :). Or just always print it (`@1`).

Reply:
I'll just not bother: I have changed the default to 2x anyway, so this is printed most of the time anyway.